augur/config.py

```diff
@@ -33,6 +33,34 @@
                     "model": "issues",
                     "repo_group_id": 0
                 },
+                {
+                    "all_focused": 1,
```

Comment:
What does this `all_focused` key actually do? I don't see it used anywhere in the code for these workers. Also, this is a separate issue, but I don't like the name `all_focused`. It's not descriptive at all and as a new user I would have no idea what it would change. The name doesn't need to convey literally everything it does, but something like `collect_all_issues` or `complete_collection` would be far more useful IMHO.

Reply:
@gabe-heim picked the name. Its for the PR and issues workers to go make sure they are getting everything done.

Reply:
So what happens when it's off? Does it just...not collect everything? Like just a subset of the data or? 
And does this worker need it? I assume so, since it's about PRs (well MRs) but I guess what I'm trying to get at is that because the name of this key is confusing, I can't tell if this is actually supposed to be here or not.

Reply:
You're asking questions about the basic design @gabe-heim has in place. I'm not doing anything new or special here. This is a switch to *ensure* that we get everything on the first run. Its been there for over two years.

Reply:
@gabe-heim : Please pull this branch and fix it, or merge your PR branch and I'll merge that here first.